docs/topics/vertx-runtime-info.adoc

```diff
@@ -0,0 +1,38 @@
+= {Vertx} Runtime Information
+
+.Certified {Vertx} Version
+
+{Vertx} version `3.4.2-3` is certified to run on OpenShift and provided as part of the Red Hat subscription.
```

Comment:
Where does this version number come from? I think I commented about this somewhere already -- the Vert.x GA version number is `3.4.2.redhat-006`.

Reply:
You're right. This version is not necessarily correct, as the changes introduced in this PR have not yet been reviewed by the Vert.x.
Thank you for catching this, I'll update momentarily:)